local.toml

```diff
@@ -187,9 +187,18 @@ title = "gitleaks config"
 	description = "Env Var"
 	regex = '''(?i)(apikey|secret|key|api|password|pass|pw|host)=[0-9a-zA-Z-_.{}]{4,120}'''
 
+[[rules]]
+	description = "Generic Username"
+	regex = '''(?i)(dbuser|user)(.{0,20})?['|"]([0-9a-zA-Z-_\/+!{}/=]{4,120})['|"]'''
+	tags = ["key", "username", "generic"]
+	[[rules.whitelist]]
+		description = "A username in a terraform file is not a leak"
+		regex = '''\w+?username\w+='''
+		file = 	'''\.tf$'''
+
 [[rules]]
 	description = "Generic Credential"
-	regex = '''(?i)(dbpasswd|dbuser|dbname|dbhost|api_key|apikey|secret|key|api|password|user|guid|hostname|pw|auth)(.{0,20})?['|"]([0-9a-zA-Z-_\/+!{}/=]{4,120})['|"]'''
+	regex = '''(?i)(dbpasswd|dbname|dbhost|api_key|apikey|secret|key|api|password|guid|hostname|pw|auth)(.{0,20})?['|"]([0-9a-zA-Z-_\/+!{}/=]{4,120})['|"]'''
```

Comment:
What's your thoughts on adding `port`? I'll get that flagged sometimes when it's set for local dev or testing defaults.

Reply:
We removed Port because it's not CUI in https://github.com/cloud-gov/caulking/pull/21/files#diff-d824368d26096c4070d4e8890635bdcbL173-L179 and for issue #12